desktop-exporter/app/components/waterfall-view/duration-bar.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Box, Flex, useColorModeValue } from "@chakra-ui/react";
+
+import { NsFromString, TraceTimeAttributes } from "../../utils/duration";
+import { EventData } from "../../types/api-types";
+
+type DurationBarProps = {
+  events: EventData[];
```

Comment:
You probably want to pass the whole `spanData` down instead of just the events since you will need it if you ever want to color the bar based on span values.